calico_containers/pycalico/datastore.py

```diff
@@ -655,7 +657,142 @@ def profile_exists(self, name):
             return True
 
     @handle_errors
-    def create_profile(self, name, rules=None):
+    def policy_exists(self, tier_name, policy_name):
+        """
+        Check if a policy exists.
+
+        :param tier_name: The name of the tier in which to search 
+        for this policy.
+        :param policy_name: The name of the policy to search for.
+        :return: True if the profile exists, false otherwise.
+        """
+        profile_path = POLICY_PATH % {"tier_name": tier_name,
+                                      "policy_name": policy_name}
+        try:
+            _ = self.etcd_client.read(profile_path)
+        except EtcdKeyNotFound:
+            return False
+        else:
+            return True
+
+    def set_policy_tier_metadata(self, tier_name, order):
```

Comment:
There's no get_policy_tier_metadata(). Maybe it's OK, but it feels a bit strange